internal/cmd/grants.go

```diff
@@ -222,6 +222,15 @@ func addGrant(cli *CLI, cmdOptions grantsCmdOptions) error {
 
 	cli.Output("Access granted!")
 
+	// Warn user if destination is kubernetes that role is not validated
+	if cmdOptions.Destination != models.InternalInfraProviderName {
```

Comment:
Not sure if we should output this every time a user runs this command. Maybe we could throw the information about roles in the `grants add` help section for now?

Reply:
I was attempting to give feedback to the user when it wasn't one of the four mentioned in our docs - "hey, this is not one of the defaults we talked about, so make sure you meant this"

if I add it in grants add, then I believe message might look like:
`don't recognize this grant, for more information, see grants add --help"